pom.xml

```diff
@@ -39,6 +39,13 @@
         <artifactId>maven-plugin</artifactId>
         <version>3.1.2</version>
         <optional>true</optional>
+        <exclusions>
+            <exclusion>
+                <!-- to avoid RequireUpperBoundDeps with jenkins-test-harness -->
+                <groupId>commons-net</groupId>
+                <artifactId>commons-net</artifactId>
+            </exclusion>
+        </exclusions>
```

Comment:
Required to avoid
```
[WARNING] Rule 5: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for commons-net:commons-net:3.5 paths to dependency are:
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.main:maven-plugin:3.1.2
    +-org.apache.maven.wagon:wagon-ftp:3.0.0
      +-commons-net:commons-net:3.5
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-test-harness:2.49
    +-org.jenkins-ci.main:jenkins-test-harness-htmlunit:2.31-2
      +-commons-net:commons-net:3.6
```